_docs/Instructions-Compile.md

```diff
@@ -1,58 +1,71 @@
+
+### Required Tools (for Mac)
+
+- XCode v15.0.1+
+  - Check or set `Command Line Tools` location
+  - **Preferences** | **Locations** | **Command Line Tools**
+- Visual Studio for Mac 2022 (EOL - April 2024)
+  - Enable .NET 8 SDK in **Preferences** | **Other** | **Preview Features**
+- [Azurite](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio,blob-storage) (For Docker or Visual Studio Code)
 
 ## The F5 Experience
 
 ### Web API
 
-1. Clone the repository `git clone https://github.com/SSWConsulting/SSW.Rewards.git`
-1. Open the `API` Project within this repo
-1. Grab connection strings from an existing dev to create local `secrets.json` file (https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.0&tabs=windows)
-1. Install / Run `Azure Storage Emulator` (Run the .exe or use the "start" command)
-1. Install / Run [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/#overview)
-1. Start Explorer (**On First Install** - See steps below)
+1. Clone the repository 
+   1. `git clone https://github.com/SSWConsulting/SSW.Rewards.API.git`
+3. Grab connection strings from an existing dev to create local `secrets.json` file (https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.0&tabs=windows)
+4. Install / Run `Azure Storage Emulator` (Run the .exe or use the "start" command)
+5. Install / Run [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/#overview)
+6. Start Explorer (**On First Install** - See steps below)
    1. Sign in using the "Subscription" option
-   1. Select the "Azure" option
-   1. Next (Sign in with your account that has access to ssw1 DevOps)
-1. Inside the Explorer - **Local & Attached** | **Storage Accounts** | **Emulator** | **Blob Containers**
-1. Create a **new container** called `profile`
-1. Copy over all files in `SeedData/profiles` (images + .xlsx) into the newly created `profile` container
-1. Press f5
-1. Test that you can access the swagger docs @ `https://localhost:5001/swagger/`
-1. **(Optional)** - If you need to run the api and mobile app locally. Set up an [ngrok](https://ngrok.com/) account (This is free and signing up allows 24hour access instead of the 2hours without an account)
+   2. Select the "Azure" option
+   3. Next (Sign in with your account that has access to ssw1 DevOps)
+7. Inside the Explorer - **Local & Attached** | **Storage Accounts** | **Emulator** | **Blob Containers**
+8. Create a **new container** called `profile`
+9.  Copy over all files in `SeedData/profiles` (images + .xlsx) into the newly created `profile` container
+10. Press f5
+11. Test that you can access the swagger docs @ `https://localhost:5001/swagger/`
+12. **(Optional)** - If you need to run the api and mobile app locally. Set up an [ngrok](https://ngrok.com/) account (This is free and signing up allows 24hour access instead of the 2hours without an account)
```

Comment:
Dev tunnels might be the better recommendation now? https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started

Reply:
We have a rule on this. It seems like Dev Tunnels is the new recommendation, NGrok second.
https://ssw.com.au/rules/port-forwarding/

Reply:
I've updated with DevTunnels :)